### ISSUE TITLE
Feature Request: Attach Note

### ISSUE THREAD

**Issue by dmnyc** [damus-io/damus#2377]
The current process of linking a note to a post requires the user to enter "nostr:" then paste the note ID. Damus also make this a little more difficult as it adds a line break before every paste so the user must know to delete the break so the note ID is directly after the colon.

Proposed solution is to add a button to attach a note, similar to attaching an image, but would let the user paste in the note ID and Damus would add the appropriate tag to the note so that it appears like a quoted note.

**Comment by alltheseas** [comment]
Can you share a screenshot of the behavior when:
1) you simply paste a noteID and post a note
2) you paste a noteID immediately after AT symbol and post a note

**Comment by dmnyc** [comment]
I never just paste the note ID by itself. Maybe I didn't realize that was the easiest way to do it. It's not explained in the app so I've been using nostr:note to ensure cross client compatibility. 

Typing nostr: and pasting adds a line break. Typing @ does not but I know some clients can't read notes linked that way. 

https://github.com/user-attachments/assets/07b57be2-e29e-473f-895f-5163a0d0ecee

**Comment by alltheseas** [comment]
> Typing @ does not but I know some clients can't read notes linked that way.

Can you share:
1) which app and method you use to post a noteID
2) noteID
3) which other app fails to parse the quoted noteID

**Comment by dmnyc** [comment]
1. Usually post with Damus on my own account, but because I manage multiple
accounts I also need to use Nostur as it’s the only iOS client I can use
with a -user option. I’ve been using @ since the start so I sometimes
forget out of habit that it’s not needed anymore. It’s complicated to
remember which apps are compatible with certain syntax!

2. Any note id, it doesn’t matter

3. Primal still has trouble with @, but only when posted using Nostur.
Notes created using Damus produced the intended result.

I just did some trial and error testing on a disposable npub and figured
this out:

https://primal.net/e/note1tdf2dy598dvtnpjaf34kg3e275m0xdlckyryz3cp54336vv8hmwq8lqrsm

**Comment by alltheseas** [comment]
Thanks for the detailed feedback @dmnyc - I'll bring to team primal's attention to improve interoperability. 

As regards to the attaching a note feature, my gut feel is that this has Microsoft Outlook vibes. I could be way wrong, and @robagreda and @ericholguin may be able to offer a superior UX perspective.

**Comment by dmnyc** [comment]
The use case is for accounts like npub1q6kncquzxqfwragm22ac3xj4x77f3z8gafd0de6tw0nsl99lweeswmn6lk which is similar to how Nostr report used to work. Display a list of notes in a scrolling feed. 

What if it showed the first one with a Read more > link below that slides the screen to the right to display them all in a separate view pane, so there's no timeline clutter?

**Comment by alltheseas** [comment]
> The use case is for accounts like npub1q6kncquzxqfwragm22ac3xj4x77f3z8gafd0de6tw0nsl99lweeswmn6lk which is similar to how Nostr report used to work. Display a list of notes in a scrolling feed.
> 
> What if it showed the first one with a Read more > link below that slides the screen to the right to display them all in a separate view pane, so there's no timeline clutter?

is this use case described by https://github.com/damus-io/damus/issues/2378?

**Comment by alltheseas** [comment]
Lmk if your request is different from than the one described in https://github.com/damus-io/damus/issues/2378

**Comment by dmnyc** [comment]
Yes they are different. #2378 refers to the way multiple links notes are displayed. This ticket refers to a UX improvement to allow users to add noteIDs as attachment.

**Comment by alltheseas** [comment]
> This ticket refers to a UX improvement to allow users to add noteIDs as attachment.

Can you provide a mockup of what exactly you envision? It is not clear to me how attachment fits in with neventID. Usually I associate attachment as a file (e.g. pfp.jpeg), whereas notes/nevents are in my experience using nostr apps not stored as stand alone files.
